app/views/videos/show.html.slim

```diff
@@ -27,12 +27,25 @@
     /   a href="#"
     /     strong mostrar mais
   .watch-also
-    .grid-container
-      - unless @video.category.nil?
+    - unless @video.series.nil?
+      .text-center
+        = link_to "Voltar para a lista de episódios da série", @video.series, class: "button success"
+        .grid-container.text-left
```

Comment:
Create a partial for this block and pass it a collection of videos, so you can keep it DRY between series and category videos.

http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials
`3.4.5 Rendering Collections`

Reply:
On the partial, check if the collection is empty and don't render anything if it is. Right now it is displaying `Assista também...` with no videos.